src/row/row.ts

```diff
@@ -0,0 +1,62 @@
+import {html, LitElement, nothing} from "lit";
+import {property, state} from "lit/decorators.js";
+import '../divider';
+
+export class Row extends LitElement {
+  @property({ reflect: true }) size: "standard" | "compact" = "standard";
+
+  @property({ reflect: true, type: Boolean }) divider: boolean = false;
+
+  @property({ reflect: true }) title: string = '';
+
+  @property({ reflect: true }) subtitle?: string;
+
```

Comment:
I think we can support the clickable state too according to the design. and add the `chevron` icon if the `@click` event is defined.